layout.deskintl.h

```diff
@@ -53,13 +64,17 @@ static struct key keys_full[], keys_special[],
   keys_compose_l[], keys_compose_z[], keys_compose_x[], keys_compose_c[],
   keys_compose_v[], keys_compose_b[], keys_compose_n[], keys_compose_m[],
   keys_compose_math[], keys_compose_punctuation[], keys_compose_bracket[],
-  keys_index[];
+  keys_compose_cyr_i[], keys_compose_cyr_j[], keys_compose_cyr_e[],
+  keys_compose_cyr_u[], keys_compose_cyr_l[], keys_compose_cyr_n[],
+  keys_compose_cyr_tse[], keys_compose_cyr_che[], keys_compose_cyr_g[],
+  keys_compose_cyr_k[],   keys_index[];
```

Comment:
A minor adjustment: Replace the three spaces before `keys_index[]` with one space.

Reply:
thanks! applied